lib/src/test/java/com/team2813/lib2813/util/InputValidationTest.java

```diff
@@ -1,38 +1,35 @@
 package com.team2813.lib2813.util;
 
-import static org.junit.Assert.assertEquals;
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth.assertWithMessage;
 import static org.junit.Assert.assertThrows;
 
-import com.team2813.lib2813.theories.Between;
-import org.junit.experimental.theories.Theories;
-import org.junit.experimental.theories.Theory;
-import org.junit.runner.RunWith;
+import org.junit.Test;
 
-@RunWith(Theories.class)
 public class InputValidationTest {
-  private static <T> RuntimeException outOfBounds(T val) {
-    return new RuntimeException("Value was out of bounds");
-  }
-
-  @Theory
-  public void invalidCanID(
-      @Between(first = -50, last = -1) @Between(first = 63, last = 100) int canID) {
-    InvalidCanIdException expected = new InvalidCanIdException(canID);
-    InvalidCanIdException actual =
-        assertThrows(InvalidCanIdException.class, () -> InputValidation.checkCanId(canID));
-    assertEquals("Messages were not identical", expected, actual);
-  }
-
-  @Theory
-  public void genericInvalidTest(@Between(first = 0, last = 4) int unused) {
-    Integer min = Integer.valueOf(0);
-    Integer actual = Integer.valueOf(unused);
-    Integer max = Integer.valueOf(4);
-    InputValidation.checkBounds(min, max, actual, InputValidationTest::outOfBounds);
-  }
+  // Tests for the `InputValidation.checkCanId(...)` method.
+  public static class checkCanIdTest {
+    @Test
+    public void invalidCanIdTest() {
+      // Can IDs can only valid in the range [0, 62].
+      int invalidCanIds[] = {-50, -1, 63, 100};
```

Comment:
This used the c-style way to declare an array. In java the `[]` is part of the type:

```suggestion
      int[] invalidCanIds = {-50, -1, 63, 100};
```

Reply:
Done.